postman_collection/StackOverflowClone - API.postman_collection.json

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
@fabioalmeida100 I think the name of the file can just be `API.postman_collection.json`, the StackOverflowClone part can be removed

Reply:
Done